src/webserver.cc

```diff
@@ -82,12 +84,17 @@ void Webserver::processConnection(int threadIndex, tcp::socket& socket) {
         }
     }
     std::string response = processRawRequest(req);
+    if (response == "") {
+        std::cout << "Error processing request\n";
```

Comment:
Boiler plate code. Maybe could be converted into a separate function call, for here and in the methods above.